docs/tutorials/istio.md

```diff
@@ -0,0 +1,28 @@
+# Configure Istio Service Mesh
+
+Integrating Istio's service mesh with Coder's Ingress enables powerful traffic management, security, and observability capabilities. By placing Coder's workspace traffic behind Istio's intelligent proxy layer, you can implement access controls, encrypt service-to-service communication, and gain visibility into your workspace network patterns. This guide walks through the process of configuring Istio alongside Coder's existing ingress controller, ensuring that developer workspaces remain accessible while benefiting from Istio's comprehensive service mesh features.
+
+Before proceeding, ensure you have a running Kubernetes cluster with both Coder and Istio installed, and that you have administrative access to configure both systems. Once you have access to your Coder cluster, apply the following manifest:
```

Comment:
```suggestion
Ensure you have a running Kubernetes cluster with both Coder and Istio installed, and that you have administrative access to configure both systems.

Once you have access to your Coder cluster, apply the following manifest:
```

Reply:
Istio is theoretically platform-independent but in practice most folks use it with Kubernetes. We can just state this assumption outright.